CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - queue-utube handling (#85)
 - Master discovery (#113)
 - SQL support (#62)
+- Add public API with a request object for Select/Update/Upstream (#126)
```

Comment:
There is a word "Added" above, seems the "Add" here is unnecessary.

Reply:
Thank you, fixed.